hybrid/dispatch/power_storage/one_cycle_baseload_battery_dispatch_heuristic.py

```diff
@@ -0,0 +1,158 @@
+from typing import Tuple
+import pyomo.environ as pyomo
+from pyomo.environ import units as u
+
+import PySAM.BatteryStateful as BatteryModel
+import PySAM.Singleowner as Singleowner
+
+from hybrid.dispatch.power_storage.simple_battery_dispatch_heuristic import SimpleBatteryDispatchHeuristic
+
+
+class OneCycleBaseloadBatteryDispatchHeuristic(SimpleBatteryDispatchHeuristic):
```

Comment:
This class is mainly the `OneCycleBatteryDispatchHeuristic` we should use class inheritance to remove all of the repeated code.